README.md

```diff
@@ -131,6 +131,8 @@ This module exposes a single function `download` which takes the same arguments
   * **parquet** loads the urls and optional caption as a parquet
 * **url_col** the name of the url column for parquet and csv (default *url*)
 * **caption_col** the name of the caption column for parquet and csv (default *None*)
+* **bbox_col** the name of the bounding box column (default *None*)
+* **bbox_format** if set to *albumentations*, the bounding box has format ```[x_min, y_min, x_max, y_max]```. If set to *coco*, the bounding box has format ```[x_min, y_min, width, height]```. All are floats, relative to the height and width of the image (default *albumentations*)
```

Comment:
what do you think about normalizing this (pick one format and convert to the other when needed) ?

Reply:
See my newest commit on this. I believe this has to be pushed towards the user, since the bounding box format is user defined (based on what is inputted in the parquet/csv file).

After thinking this further, I now set this to None, and if bbox_col is also set, then the format also needs to be explicitly set. In the final parquet files the format is kept unchanged, but internally all formats are changed to a common one for further processing.

Let me know if this behavior is acceptable.